cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/AbstractProgramRuntimeService.java

```diff
@@ -81,6 +82,7 @@
 public abstract class AbstractProgramRuntimeService extends AbstractIdleService implements ProgramRuntimeService {
 
   private static final Logger LOG = LoggerFactory.getLogger(AbstractProgramRuntimeService.class);
+  private static final String APPLICATION_SCOPE = "application";
```

Comment:
Not a biggie, but should it be "application" or just "app"? In REST URLs, we have /apps/, also in Metrics we use the tag name "app".